package gateway;

enum MessageId {
	LOGIN_REQUEST = 1;
	LOGIN_RESPONSE = 2;
}

enum StatusCode {
	SUCCESS = 0;
}

message LoginRequestMessage {
	optional int64 steam_id = 1;
}

message LoginResponseMessage {
	required StatusCode status_code = 1;
}
